### PR TITLE
Add analytics for wallet actions

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description
Adds analytics for claiming & sending $AUDIO

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?
nope

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

Tested request/success/failure against staging for claim & send. Verified wallet addresses were correct in analytics payloads.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/rj-claim-analytics

File: src/services/analytics.ts [hunk 1]
```diff
@@ -664,6 +670,39 @@ type RemixHide = {
   handle: string
 }
 
+type ClaimAudioRequest = {
+  eventName: Name.CLAIM_AUDIO_REQUEST
+  wallet: string
```
Comment (piazzatron) [inline src/services/analytics.ts:675]: nit: I *think* we have a wallet or address type for this somewhere, could be wrong though

File: src/store/wallet/sagas.ts [hunk 2]
```diff
@@ -47,26 +57,54 @@ function* sendAsync({
...
+        from: account?.wallet,
+        recipient: recipientWallet
+      })
+    )
   } catch (e) {
     yield put(sendFailed({ error: e.message }))
+    yield put(
+      make(Name.SEND_AUDIO_FAILURE, {
+        from: account?.wallet,
+        recipient: recipientWallet
```
Comment (piazzatron) [inline src/store/wallet/sagas.ts:71]: would it be useful to add the error message here?

**Review by piazzatron — APPROVED** [review]

File: src/store/wallet/sagas.ts (hunk 2 above)
Comment (raymondjacobson) [reply src/store/wallet/sagas.ts:71]: I dont think we can/should stuff error messages into amplitude, probably more a sentry thing

File: src/services/analytics.ts (hunk 1 above)
Comment (raymondjacobson) [reply src/services/analytics.ts:675]: nice!

**Pull request merged**